Azure-Samples/communication-services-ios-calling-hero
# Add AAD login and profile / avatar image retrieval

## Purpose
Updates the iOS calling hero app calling-ui-library branch with the UI calling library.
This PR is only to cover the log-in cases in this ticket: https://skype.visualstudio.com/SPOOL/_workitems/edit/2912002

## Does this introduce a breaking change?
```
[x] Yes
[ ] No
```

## Pull Request Type
```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Get the code, set up your own ACS and AAD services and ensure you can log in. You will also see your avatar picture if you have one, or initials if you don't.

## Other Information
This is an interim PR to a feature branch, created mainly to allow others to share some of the common layout code introduced as we are moving to code-based layout, rather than storyboards.
